src/NumbersToLetters.java

```diff
@@ -40,9 +40,6 @@ public static void main(String[] args) {
             } else {
                 System.out.println(ongeldig);
             }
-            //input van de user krijgen en gebruiken
-
-            //gegevens uitprinten
 
 
         }
```

Comment:
De arrays numeric en alphabetic staan nu midden in je while loop die zou ik voor de overzichtelijkheid en in de toekomst scope bij je andere variables zetten boven aan je main.